db/utils.js

```diff
@@ -63,12 +63,15 @@ function getReadOnlyDBConfig(readReplicas) {
     readConfig = [];
     readReplicas.forEach((replicaUrl) => {
       const dbConfObj = dbConfigObjectFromDbURL(replicaUrl);
-      readConfig.push({
-        host: dbConfObj.host,
-        port: dbConfObj.port,
-        username: dbConfObj.user,
-        password: dbConfObj.password,
-      });
+      if (dbConfObj.host && dbConfObj.port &&
```

Comment:
Can you test this locally or in a review app to see if this is the only place to do the check.? Might have to change getReadReplicas in /config/configUtil.js.